Source/WebCore/page/LocalDOMWindow.h

```diff
@@ -277,7 +277,7 @@ class LocalDOMWindow final
 
     void scrollBy(const ScrollToOptions&) const;
     void scrollBy(double x, double y) const;
-    void scrollTo(const ScrollToOptions&, ScrollClamping = ScrollClamping::Clamped, ScrollSnapPointSelectionMethod = ScrollSnapPointSelectionMethod::Closest) const;
+    void scrollTo(const ScrollToOptions&, ScrollClamping = ScrollClamping::Clamped, ScrollSnapPointSelectionMethod = ScrollSnapPointSelectionMethod::Closest, ScrollClass scrollClass = ScrollClass::ScrollTo) const;
```

Comment:
This is explicitly a `scrollTo` so I'm not sure how the parameter makes sense.

Reply:
See above.